libs/designer-ui/src/lib/editor/base/index.tsx

```diff
@@ -44,6 +44,7 @@ export type GetTokenPickerHandler = (
   editorId: string,
   labelId: string,
   tokenPickerMode?: TokenPickerMode,
+  editorType?: string,
```

Comment:
Is this being used?

Reply:
yeah its what we pass from the editor to the tokenpicker